sdk/script/account-manager.ts

```diff
@@ -0,0 +1,349 @@
+import * as base64 from "base-64";
+import Q = require("q");
+import crypto = require("crypto");
+import Promise = Q.Promise;
+import superagent = require("superagent");
+
+var packageJson = require("../../package.json");
+
+declare var fs: any;
+
+if (typeof window === "undefined") {
+    fs = require("fs");
+} else {
+    fs = {
+        createReadStream: (fileOrPath: string): void => {
+            throw new Error("Tried to call a node fs function from the browser.");
+        }
+    }
+}
+
+// Aliasing UpdateMetrics as IUpdateMetrics & CollaboratorProperties as ICollaboratorProperties to deal with TypeScript issue that removes unused imports.
+import { AccessKey, Account, App, CollaboratorMap, CollaboratorProperties as ICollaboratorProperties, Deployment, DeploymentMetrics, Package, UpdateMetrics as IUpdateMetrics } from "rest-definitions";
+export { AccessKey, Account, App, CollaboratorMap, Deployment, DeploymentMetrics, Package };
+export type UpdateMetrics = IUpdateMetrics;
+export type CollaboratorProperties = ICollaboratorProperties;
+
+export module Permissions {
+    export const Owner = "Owner";
+    export const Collaborator = "Collaborator";
+}
+
+export interface CodePushError {
+    message?: string;
+    statusCode?: number;
+}
+
+interface PackageToUpload {
+    description: string;
+    appVersion: string;
+    isMandatory: boolean;
+}
+
+interface JsonResponse {
+    header: { [headerName: string]: string };
+    body?: any;
+}
+
+// A template string tag function that URL encodes the substituted values
+function urlEncode(strings: string[], ...values: string[]): string {
+    var result = "";
+    for (var i = 0; i < strings.length; i++) {
+        result += strings[i];
+        if (i < values.length) {
+            result += encodeURIComponent(values[i]);
+        }
+    }
+
+    return result;
+}
+
+export class AccountManager {
+    private static API_VERSION: number = 2;
+
+    private _accessKey: string;
+    private _serverUrl: string;
+    private _userAgent: string;
+
+    constructor(accessKey: string, userAgent?: string, serverUrl?: string) {
+        this._accessKey = accessKey;
+        this._userAgent = userAgent;
+        this._serverUrl = serverUrl || "https://codepush.azurewebsites.net";
```

Comment:
Change this default to `https://codepush-management.azurewebsites.net` to reflect the splitting of the management server from the acquisition server.